docker-compose/study/docker-compose.override.yml

```diff
@@ -329,15 +329,15 @@ services:
     depends_on:
       - logspout
     environment:
-      - JAVA_TOOL_OPTIONS=-Xmx186m
+      - JAVA_TOOL_OPTIONS=-Xmx186m # not used by the native image
```

Comment:
We can comment the property nope ?

Reply:
I'll keep it for now as otherwise it adds some changes to do when developers want to use the JVM image. (and I also need to comment the "enviroment:" line).